Doc/howto/argparse.rst

```diff
@@ -841,6 +841,69 @@ translated messages.
 
 To translate your own strings in the :mod:`argparse` output, use :mod:`gettext`.
 
+Custom type converters
```

Comment:
We should probably indicate that custom type converters are called with a single argument, namely the argument (string) value. No context is passed around for instance.

I don't know if the docs are already explaining it but you can probably say that a more refined parsing strategy can be achieved using custom action classes where you need to subclass `__call__` and possibly `__init__` (actually, is it part of the public API? I don't remember whether it's a hack to create custom action classes).

Reply:
Good catch - added!